lib/testLauncher/SuitestLauncher.js

```diff
@@ -36,6 +37,62 @@ class SuitestLauncher {
 		});
 	}
 
+	async runTokenSession() {
+		suitest.logger.intro(launcherGreeting, version, TOKEN);
+		validateInput(TOKEN.toUpperCase(), this.ownArgv);
+
+		try {
+			/* istanbul ignore if  */
+			if (isDebugMode(this.ownArgv)) {
+				// debugging not allowed in automated session
+				throwDebugInAutomatedError();
+			}
+
+			const devices = Object.keys(this.ownArgv.presets)
+				.map((key) => {
+					return {
+						device: this.ownArgv.presets[key].device,
```

Comment:
you also need cover case when `device` and `config` properties not string, but objects that contains `deviceId` and `configId` respectively. Please fix it.